src/styles/partials/_positioning.scss

```diff
@@ -0,0 +1,13 @@
+h1, h2, h3 {
```

Comment:
These overrides are basic margins for every heading element and paragraph which were previously given by bootstrap to have a uniform styles over each page. They can be overridden in each component scss file if need be.